src/Tokenizer.php

```diff
@@ -182,13 +182,10 @@ final class Tokenizer
         'MYISAM',
         'NAMES',
         'NATURAL',
-        'NO OTHERS',
```

Comment:
I think I might not be understanding what you are doing here… shouldn't you be adding back `NO` as well as `OTHERS` to this list?

Reply:
`NO` or `OTHERS` must not be added into "reserved" keywords as both are not "reserved" by SQL/MySQL spec.

Reply:
> Make sure keywords are single upper words

Your commit message does not mention that at any point.

Reply:
The PR description does. If the commit message should be amended, please tell me what text to put there.

Reply:
We need to end up exactly with the changes as this PR does - please tell me exactly what to do, am I right you want separate commit for `'NO OTHERS'` keyword removal? Anything else?

Reply:
2 commits:
- one that removes `NO OTHERS` and explains why, and also takes a guess at why `NO OTHERS` was present in this list in the first place
- another that is about making sure we deal with single words, and also explains why that is necessary.

Reply:
Done and thank you.

Reply:
Why do you think `NO OTHERS` was present in the list in the first place? You cite MySQL grammar, but this library is not solely about MySQL, is it?

Reply:
> Why do you think `NO OTHERS` was present in the list in the first place?

See https://github.com/search?q=repo%3Amysql%2Fmysql-server+%22NO+OTHERS%22&type=code MySQL tests for usecases and https://github.com/antlr/grammars-v4/blob/4264f19a7d/sql/mysql/Oracle/MySQLParser.g4#L1208 for full grammar.

It is however not a "reserved" keyword because of two reasons. Keyword must be a single word. And must be listed as reserved in MySQL or MariaDB docs.

You cite MySQL grammar, but this library is not solely about MySQL, is it?

The jdorm library was for MySQL dialect only. In #106 many keywords will be added as the current list is far from incomplete.